sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java

```diff
@@ -84,10 +88,17 @@ public static synchronized void init(
           OptionsContainer.create(SentryAndroidOptions.class),
           options -> {
             final LoadClass classLoader = new LoadClass();
+            final boolean isTimberUpstreamAvailable =
+                classLoader.isClassAvailable(TIMBER_CLASS_NAME, options);
+            final boolean isFragmentUpstreamAvailable =
+                classLoader.isClassAvailable(FRAGMENT_CLASS_NAME, options);
             final boolean isFragmentAvailable =
-                classLoader.isClassAvailable(SENTRY_FRAGMENT_INTEGRATION_CLASS_NAME, options);
+                (isFragmentUpstreamAvailable
```

Comment:
`m`: What error message does the user get if any timber or fragment is not available? Will they know from the error message how to resolve the issue?

Reply:
It will say "Class not available: Fragment..." with a debug log level. That's fine, because it's part of the auto-install feature and it's not supposed to be always installed, there's no action for the user to take.